compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -5377,14 +5377,25 @@ OMR::Z::CodeGenerator::replaceInst(TR::Instruction* old, TR::Instruction* curr)
       }
    }
 
+// TODO: This should really be a common code generator API. Push this up to the common code generator.
```

Comment:
Please open a GitHub issue for this TODO and reference it in the code comment. Otherwise it is likely to be forgotten.